src/common/main/kotlin/tech/thatgravyboat/skyblockapi/api/events/base/EventBus.kt

```diff
@@ -83,6 +84,7 @@ class EventBus {
 
         val kotlin = method.kotlinFunction
         if (kotlin?.extensionReceiverParameter != null && McClient.isDev && Modifier.isPublic(method.modifiers)) {
+            DebugEvents.methodsToWarn.add(method)
```

Comment:
this can be dynamically added to and removed so you can possibly keep increasing this list forever depending on the registering of an event, this should not be added to after the in-game warning has been ran once or should after that point always log in chat